css/sponsor.css

```diff
@@ -1,3 +1,13 @@
+.container-fluid {
+  padding-left: 15%;
+  padding-right: 15%;
+}
+
+.stats {
+  padding-left: 5%;
+  padding-right: 5%;
+}
+
```

Comment:
all the content should have the same amount of whitespace to the left and the right -- i think slightly reducing the padding from the intro text & slightly increasing the padding to the stats to make them the same would look best, like this: 
![Screen Shot 2021-06-13 at 9 11 00 AM](https://user-images.githubusercontent.com/62481939/121808612-4c403800-cc27-11eb-8d3f-03f7720aadcc.png)